scripts/HoloFuelDataEntry/index.js

```diff
@@ -0,0 +1,310 @@
+// //////////////////////////////////////////////////////////////////////////////////// //
```

Comment:
This all looks great. Curious at higher level, what it would have looked like to orchestrate this as a series of GraphQL mutations / queries to achieve the same?  Not thinking it makes more sense now, but in the future it could be worth exploring.

Reply:
Thanks for the review and thought.  Actually, the thought to call a series of GraphQL calls was one of the very discussions that @robbiecarlton and I held at the onset of this task.  We agreed on this approach in lieu of the gql for a few reasons, including cli simplicity and direct access to the conductor, but I agree that this still is a thought to consider for the integration / scenario testing approach. :+1: